docs/modules/clients/pages/java-thin-client.adoc

```diff
@@ -0,0 +1,1126 @@
+= {client-name}
+:page-api-reference: https://docs.hazelcast.org/docs/{page-latest-supported-java-client}/javadoc
+:url-cloud-signup: https://cloud.hazelcast.com/sign-up
+:page-toclevels: 3
+:client-name: Thin Java Client
```

Comment:
Depending on result of naming and licensing discussions, we will need something like:
```suggestion
:client-name: Java Client (Standard)
:page-enterprise: true
:page-beta: true
```
We can (and should) define the variable globally via https://github.com/hazelcast/hz-docs/blob/main/docs/antora.yml

Reply:
Done